src/Vulkan/Silk.NET.Vulkan/VkOverloads.gen.cs

```diff
@@ -1720,48 +1727,55 @@ public static unsafe Result CreateImageView(this Vk thisApi, [Count(Count = 0)]
         }
 
         /// <summary>To be documented.</summary>
+        [Inject((SilkTouchStage) 6, "if (%$RESULT$% == Result.Success) { CurrentInstance = *%$PARAM(pInstance)$%; }")]
         public static unsafe Result CreateInstance(this Vk thisApi, [Count(Count = 0), Flow(FlowDirection.In)] InstanceCreateInfo* pCreateInfo, [Count(Count = 0), Flow(FlowDirection.In)] AllocationCallbacks* pAllocator, [Count(Count = 0), Flow(FlowDirection.Out)] Span<Instance> pInstance)
         {
             // SpanOverloader
             return thisApi.CreateInstance(pCreateInfo, pAllocator, out pInstance.GetPinnableReference());
         }
 
         /// <summary>To be documented.</summary>
+        [Inject((SilkTouchStage) 6, "if (%$RESULT$% == Result.Success) { CurrentInstance = *%$PARAM(pInstance)$%; }")]
         public static unsafe Result CreateInstance(this Vk thisApi, [Count(Count = 0), Flow(FlowDirection.In)] InstanceCreateInfo* pCreateInfo, [Count(Count = 0), Flow(FlowDirection.In)] ReadOnlySpan<AllocationCallbacks> pAllocator, [Count(Count = 0), Flow(FlowDirection.Out)] Instance* pInstance)
         {
             // SpanOverloader
             return thisApi.CreateInstance(pCreateInfo, in pAllocator.GetPinnableReference(), pInstance);
         }
 
         /// <summary>To be documented.</summary>
+        [Inject((SilkTouchStage) 6, "if (%$RESULT$% == Result.Success) { CurrentInstance = *%$PARAM(pInstance)$%; }")]
         public static unsafe Result CreateInstance(this Vk thisApi, [Count(Count = 0), Flow(FlowDirection.In)] InstanceCreateInfo* pCreateInfo, [Count(Count = 0), Flow(FlowDirection.In)] ReadOnlySpan<AllocationCallbacks> pAllocator, [Count(Count = 0), Flow(FlowDirection.Out)] Span<Instance> pInstance)
         {
             // SpanOverloader
             return thisApi.CreateInstance(pCreateInfo, in pAllocator.GetPinnableReference(), out pInstance.GetPinnableReference());
         }
 
         /// <summary>To be documented.</summary>
+        [Inject((SilkTouchStage) 6, "if (%$RESULT$% == Result.Success) { CurrentInstance = *%$PARAM(pInstance)$%; }")]
         public static unsafe Result CreateInstance(this Vk thisApi, [Count(Count = 0), Flow(FlowDirection.In)] ReadOnlySpan<InstanceCreateInfo> pCreateInfo, [Count(Count = 0), Flow(FlowDirection.In)] AllocationCallbacks* pAllocator, [Count(Count = 0), Flow(FlowDirection.Out)] Instance* pInstance)
         {
             // SpanOverloader
             return thisApi.CreateInstance(in pCreateInfo.GetPinnableReference(), pAllocator, pInstance);
         }
 
         /// <summary>To be documented.</summary>
+        [Inject((SilkTouchStage) 6, "if (%$RESULT$% == Result.Success) { CurrentInstance = *%$PARAM(pInstance)$%; }")]
```

Comment:
I guess these don't hurt?

Reply:
Pffffft haha, didn't realise it'd add them too.

Let's leave them in for posterity and one day revisit it :D